src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/integration/resource/VideoLinkBookingIntegrationTest.kt

```diff
@@ -247,12 +250,11 @@ class VideoLinkBookingIntegrationTest : SqsIntegrationTestBase() {
       appointments() hasSize 1
     }
 
-    // There should be 3 notifications - 1 user email, 1 court email and 1 prison email
-    val notifications = notificationRepository.findAll().also { it hasSize 3 }
+    // There should be 2 notifications - 1 court email and 1 prison email
```

Comment:
This would have been sending user email to the prison user.  Not sure this is correct.  Can add back in. The prison will get an email anyway.